package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "fastify-get-head",
+  "version": "1.0.0",
+  "description": "Set a HEAD route handler for each GET handler previously registered",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "scripts": {
+    "test": "tap test/*.test.js && npm run typescript",
+    "test-ci": "tap --cov test/*.test.js && npm run typescript",
+    "lint": "standard | snazzy",
+    "lint-ci": "standard",
+    "typescript": "tsd"
+  },
+  "keywords": [
+    "fastify",
+    "fastify-get-head",
+    "http",
+    "get",
+    "head"
+  ],
+  "author": "Carlos Fuentes <carlos.fuentes.rios.95@gmail.com>",
+  "license": "MIT",
+  "devDependencies": {
+    "@types/node": "^14.10.3",
+    "fastify": "^3.7.0",
+    "pre-commit": "^1.2.2",
+    "snazzy": "^8.0.0",
+    "standard": "^15.0.0",
```

Comment:
update this

Reply:
Done, please see [bb78a12](https://github.com/MetCoder95/fastify-get-head/pull/1/commits/bb78a120746ff60418d25a4e80cb7436e160ac33)